kong.conf.default

```diff
@@ -95,6 +95,14 @@
                                           # is adjusted by the `log_level`
                                           # property.
 
+#vaults = off                    # Comma-separated list of vaults this node
```

Comment:
Does this act as our "feature flag?" Is vault code inactive unless this is not empty?

Reply:
Not all code is inactive, but most of the code just runs if there are references. I am thinking about adding if statements if vaults is empty. Yes, I am thinking about using this as feature flag. Similarly we could perhaps optimize a lot of kong if no plugins are specified.